packages/genesys-spark-components/src/components/beta/gux-avatar-group/gux-avatar-group.tsx

```diff
@@ -0,0 +1,78 @@
+import { Component, h, JSX, Element, Host, Listen } from '@stencil/core';
+import { trackComponent } from '@utils/tracking/usage';
+import { logWarn } from '@utils/error/log-error';
+import { focusGroupItem } from './gux-avatar-group.service';
+
+/**
+ * @slot - slot for gux-avatar-group-item components
+ */
+@Component({
+  styleUrl: 'gux-avatar-group.scss',
+  tag: 'gux-avatar-group-beta',
+  shadow: { delegatesFocus: true }
+})
+export class GuxAvatarGroup {
+  @Element()
+  root: HTMLElement;
+
+  async componentWillLoad(): Promise<void> {
+    trackComponent(this.root);
+    if (this.root.querySelectorAll('gux-avatar-group-item-beta').length === 0) {
+      logWarn(
+        this.root,
+        'gux-avatar-group-beta: No gux-avatar-group-item-beta tags found in slot. Please add gux-avatar-group-item-beta tags to the slot.'
+      );
+    }
+  }
+
+  componentDidLoad(): void {
+    this.focusFirstGroupItem();
+  }
+
+  private focusFirstGroupItem(): void {
+    const groupItems = this.getRootChildren(
+      this.root
+    ) as HTMLGuxAvatarGroupItemBetaElement[];
+    if (groupItems.length > 0) {
+      const firstGroupItem = groupItems[0] as Element;
+      focusGroupItem(firstGroupItem);
+    }
+  }
+
+  private getRootChildren(root: HTMLElement): Element[] {
+    const slot = root.querySelector('slot');
+
+    return slot ? slot.assignedElements() : Array.from(root.children);
+  }
+
+  @Listen('mouseover')
+  onMouseOver(event: MouseEvent): void {
+    const target = event.target as HTMLElement;
+    const children = Array.from(
+      this.root.querySelectorAll('gux-avatar-group-item-beta')
+    ) as HTMLGuxAvatarGroupItemBetaElement[];
+    const focusedChildren = children.filter(child =>
+      child.matches(':focus-within')
+    );
```

Comment:
if there can only ever be one child with focus within you can use `find` instead of `filter`